src/druidry/data_source.py

```diff
@@ -158,7 +158,19 @@ def equality_filter(f):
     if left['type'] == 'field' and right['type'] == 'field':
         result = ColumnComparisonFilter(dimensions=[left['field'], right['field']])
     elif left['type'] == 'field' and right['type'] == 'value':
-        result = SelectorFilter(dimension=left['field'], value=right['value'])
+        fields = [
+            SelectorFilter(
+                dimension=left['field'],
+                value=value)
+            for value in right['value']
+        ]
+        if len(fields) == 0:
+            return None
+        if len(fields) == 1:
+            result = fields[0]
+        else:
+            result = OrFilter(fields=fields)
```

Comment:
Does this have to be an OrFilter if there are multiple Selector Filters?

Reply:
if multiple filter comes we need to consider all of them, so OrFilter